debugd/metricbeat/templates/metricbeat.yml

```
@@ -0,0 +1,63 @@
+http.port: {{ .Port }}
+
+output.logstash:
+  hosts: ["{{ .LogstashHost }}"]
+
+output.console:
+  enabled: false
+
+logging:
+  to_files: false
+  metrics.enabled: false
+  level: warning
+
+timestamp.precision: nanosecond
+
+# https://www.elastic.co/guide/en/beats/metricbeat/current/configuration-path.html#_system_hostfs
+system.hostfs: "/hostfs"
+
+metricbeat.modules:
+  {{ if .CollectSystemMetrics }}
+  - module: system
+    enabled: true
+    metricsets:
+      - cpu             # CPU usage
+      - load            # CPU load averages
+      - memory          # Memory usage
+      - network         # Network IO
+      - process         # Per process metrics
+      - process_summary # Process summary
+      #- uptime          # System Uptime
+      #- socket_summary  # Socket summary
+      #- core           # Per CPU core usage
+      - diskio         # Disk IO
+      - filesystem     # File system usage for each mountpoint
+      - fsstat         # File system summary metrics
+      #- raid           # Raid
+      #- socket         # Sockets and connection info (linux only)
+      #- service        # systemd service information
+    cpu.metrics:  ["percentages","normalized_percentages"]
+    period: 10s
```

```suggestion
    period: 60s
```

Prometheus seems to default to 1m. I guess that would be fine for us as well?
https://prometheus.io/docs/prometheus/latest/configuration/configuration/